feat: Auto generate RTL styles using rtlcss

surajshetty3416 opened the pull request:
Instead of fixing styles for RTL one-by-one [as we were doing here](https://github.com/frappe/frappe/pull/13195), let [rtlcss](https://github.com/MohammadYounes/rtlcss) auto-generate RTL styles for us.

## Screenshots:

### Before:

<img width="1603" alt="Screenshot 2021-05-17 at 4 59 42 PM" src="https://user-images.githubusercontent.com/19775888/118482588-8f93ad80-b732-11eb-82db-37c47d880f86.png">
<img width="1739" alt="Screenshot 2021-05-17 at 4 59 30 PM" src="https://user-images.githubusercontent.com/19775888/118482508-725edf00-b732-11eb-8a9f-c14cb6b208b6.png">


### Now:

<img width="1423" alt="Screenshot 2021-06-28 at 9 35 54 AM" src="https://user-images.githubusercontent.com/13928957/123579407-75e78a80-d7f5-11eb-8bcb-e0e9a2ad7e41.png">
<img width="1430" alt="Screenshot 2021-06-28 at 9 37 18 AM" src="https://user-images.githubusercontent.com/13928957/123579414-7849e480-d7f5-11eb-826c-23bf8295613e.png">
<img width="1421" alt="Screenshot 2021-06-28 at 9 41 12 AM" src="https://user-images.githubusercontent.com/13928957/123579416-7a13a800-d7f5-11eb-9e24-0c3d8114fbd7.png">
<img width="1428" alt="Screenshot 2021-06-28 at 10 15 42 AM" src="https://user-images.githubusercontent.com/13928957/123583098-abdc3d00-d7fc-11eb-9ac8-7e335dae4905.png">
<img width="616" alt="Screenshot 2021-06-28 at 9 44 18 AM" src="https://user-images.githubusercontent.com/13928957/123579424-7c760200-d7f5-11eb-9d41-a8e000c238c8.png">


Fixes:
https://github.com/frappe/frappe/issues/13259
https://github.com/frappe/frappe/pull/13195#issuecomment-850252751

> no-docs

surajshetty3416 committed:
feat: Auto generate RTL styles using rtlcss

https://github.com/MohammadYounes/rtlcss

surajshetty3416 committed:
refactor: Remove RTL style files

surajshetty3416 committed:
refactor: Remove unused build.json

surajshetty3416 committed:
feat: Select css-rtl folder if language is rtl

surajshetty3416 committed:
fix: RTL for print views

surajshetty3416 committed:
refactor: Rename dir to layout_direction

- For readability

surajshetty3416 committed:
fix: Add language to print_template

surajshetty3416 committed:
Merge branch 'develop' of https://github.com/frappe/frappe into auto-generate-rtl-style

surajshetty3416 committed:
feat: Execute separate build command for style files

- Execute style build twice (one for LTR and one for RTL) because rtlcss is dependant on comments and esbuild strips all comments so we cannot run rtlcss on built css files.

surajshetty3416 committed:
Merge branch 'develop' of https://github.com/frappe/frappe into auto-generate-rtl-style

surajshetty3416 committed:
fix: Build asset file from results sequentially

surajshetty3416 committed:
fix: Use different key for RTL assets

surajshetty3416 committed:
fix: Breadcrumb direction for RTL

surajshetty3416 committed:
fix: Add layout direction to pdf template

surajshetty3416 committed:
fix: Add RTL control directive to override dropdown style in RTL mode

amutaher commented:
Good work,
still some issue in dashboard RTL like this:

![image](https://user-images.githubusercontent.com/45880595/125209478-acb3ab00-e2a1-11eb-8171-adac15ea5dd6.png)

surajshetty3416 committed:
Merge branch 'develop' into auto-generate-rtl-style

surajshetty3416 committed:
fix: Keep charts as it is

- Frappe Charts do not support RTL and rtlcss does not work on SVG...
Setting direction as ltr to keep charts as it is

surajshetty3416 committed:
Merge branch 'develop' of https://github.com/frappe/frappe into auto-generate-rtl-style
